portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,14 +32,29 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = new Gson().toJson(com);
-     // Convert the server stats to JSON
-    System.out.println(json); //cloud shell
-    // Send the JSON as the response
+    // String json = new Gson().toJson(com);
+    //  // Convert the server stats to JSON
+    // System.out.println(json); //cloud shell
+    // // Send the JSON as the response
+    // response.setContentType("application/json;");
+    // response.getWriter().println(json); //web page
+    // // console.log(json);
+    Query query = new Query("Comment");
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<String> comments = new ArrayList<>();
```

Comment:
With addition of `comments` ArrayList I don't think you need `com` ArrayList on line 21 anymore. Could you move it inside doPost()?